src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -44,143 +43,142 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
         }
       }
     } ~
-    (pathPrefix("resources" / "v1") | pathPrefix("resource")) {
-      requireUserInfo { userInfo =>
-        pathPrefix(Segment) { resourceTypeName =>
-          withResourceType(ResourceTypeName(resourceTypeName)) { resourceType =>
-            pathEndOrSingleSlash {
-              getUserPoliciesForResourceType(resourceType, userInfo) ~
-              postResource(resourceType, userInfo)
-            } ~
-            pathPrefix(Segment) { resourceId =>
-              val resource = FullyQualifiedResourceId(resourceType.name, ResourceId(resourceId))
-
+      (pathPrefix("resources" / "v1") | pathPrefix("resource")) {
+        requireUserInfo { userInfo =>
+          pathPrefix(Segment) { resourceTypeName =>
+            withResourceType(ResourceTypeName(resourceTypeName)) { resourceType =>
               pathEndOrSingleSlash {
-                deleteResource(resource, userInfo) ~
-                postDefaultResource(resourceType, resource, userInfo)
-              } ~
-              pathPrefix("action") {
-                pathPrefix(Segment) { action =>
-                  pathEndOrSingleSlash {
-                    getActionPermissionForUser(resource, userInfo, action)
-                  }
-                }
-              } ~
-              pathPrefix("authDomain") {
-                pathEndOrSingleSlash {
-                  getResourceAuthDomain(resource, userInfo)
-                }
+                getUserPoliciesForResourceType(resourceType, userInfo) ~
+                  postResource(resourceType, userInfo)
```

Comment:
Why the tab for `postResource` here (and with other functions below)? These are just a get and post to the same url, so I'm thinking they should be tabbed in the same amount. Is that just a setting in scalafmt? It seemed to do well with `ManagedGroupRoutes`, but doesn't seem to like `ResourceRoutes` as much. I guess maybe we need to rework the routes files to be more consistent for scalafmt to be able to format them consistently.